packages/blocks-editor/src/controls/object/responsive-media-control.tsx

```
@@ -0,0 +1,123 @@
+
+	return show ? (
+		<>
+			<div style={{ display: "flex", gap: "16px", marginBottom: "16px" }}>
+				{objectKeys(screenSizes).map((screenSize) => (
+					<Button
+						variant={selectedSize === screenSize ? "primary" : "secondary"}
+						icon={<Icon icon={screenSizes[screenSize].icon} />}
+						onClick={() => setSelectedSize(screenSize)}
+						size="small"
+						className="responsive-screen-select-button"
+					>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)